client/src/utils/api.js

```diff
@@ -9,6 +9,12 @@ const defaultCallback = { onSuccess: identity }
 export default function callApi(callDescriptor, callbacks = {}) {
   const authEmail = Cookies.get('X-User-Email')
   const authToken = Cookies.get('X-User-Token')
+  const windowUrl = window.location.hostname
+  const protocol = window.location.protocol // eslint-disable-line
+  const apiUrl =
+    windowUrl === 'localhost'
+      ? 'api.roadmaps.lvh.me:5000/api/v1'
+      : window.location.hostname
```

Comment:
Will just the hostname retrieved from window.location.hostname be enough to include the /api/v1 portion? Doesn't matter with the above linkedin endpoint, but it will matter with the rest of our API here.